packages/itwin/breakdown-trees/package.json

```diff
@@ -58,22 +58,15 @@
     "@itwin/appui-layout-react": "^3.0.0",
     "@itwin/measure-tools-react": "workspace:*",
     "@testing-library/react": "^11.2.6",
-    "@types/i18next": "^8.4.2",
     "@types/react": "^17.0.19",
-    "@types/react-dom": "^17.0.9",
```

Comment:
im surprised you were able to remove this, this seem like a dep we usually always have in react projs

Reply:
I did test the package with these changes and everything worked. However, I am not 100% confident that we don't need this package, therefore, I returned it.